Dockerfile

```diff
@@ -1,75 +1,63 @@
-FROM ubuntu:20.04
+FROM ubuntu:24.04
 
 ENV WORKSPACE_DIR /root
-ENV FIRMWARE_DIR ${WORKSPACE_DIR}/Firmware
+ENV FIRMWARE_DIR ${WORKSPACE_DIR}/px4
 ENV SITL_RTSP_PROXY ${WORKSPACE_DIR}/sitl_rtsp_proxy
 
 ENV DEBIAN_FRONTEND=noninteractive DEBCONF_NONINTERACTIVE_SEEN=true
 ENV DISPLAY :99
 ENV LANG C.UTF-8
 
 RUN apt-get update && \
-    apt-get install -y wget lsb-release && \
-    apt-get -y autoremove && \
-    apt-get clean autoclean && \
-    rm -rf /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/*
-
-RUN wget https://packages.osrfoundation.org/gazebo.gpg -O /usr/share/keyrings/pkgs-osrf-archive-keyring.gpg && \
-    echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/pkgs-osrf-archive-keyring.gpg] http://packages.osrfoundation.org/gazebo/ubuntu-stable $(lsb_release -cs) main" | tee /etc/apt/sources.list.d/gazebo-stable.list > /dev/null
-
-
-RUN apt-get update && \
-    apt-get install -y bc \
```

Comment:
PX4 still needs `bc` pretty sure:

https://github.com/PX4/PX4-Autopilot/blob/c06ba93175782e5a90a83017fb2f92bb69da58f8/ROMFS/px4fmu_common/init.d-posix/rcS#L199

Reply:
Oh, for the speed factor. Maybe it explains why it didn't fail in my tests :innocent:. Adding it back!